docs/gen-ai/gen-ai-metrics.md

```diff
@@ -0,0 +1,158 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Generative AI metrics
+--->
+
+# Semantic Conventions for Generative AI Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+The conventions described in this section are specific to Generative AI
+applications.
+
+**Disclaimer:** These are initial Generative AI client metric instruments
+and attributes but more may be added in the future.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Generative AI Operations](#generative-ai-operations)
+  - [Metric: `gen_ai.tokens.usage`](#metric-gen_aitokensusage)
+  - [Metric: `gen_ai.operation.duration`](#metric-gen_aioperationduration)
+
+<!-- tocstop -->
+
+## Generative AI Operations
+
+The following metric instruments describe Generative AI operations. An
```

Comment:
@lmolkova I've changed this to Generative AI Operations to be consistent with wording in the metrics. I also attempted an explanation of "operation". Your feedback please. :)